index.js

```diff
@@ -39,35 +40,23 @@ module.exports = function (content) {
     }
 
     var filePath = prefix + resource.slice(relativeToIndex + relativeTo.length); // get the base path
-    var html;
+    var html, match = content.match(/^module\.exports.+?(".*")/);
```

Comment:
I have searched through those commonly used loader, and they all produce string with double quotes inside.
So, it could be concluded that there is common sense among developers with double quotes.